modules/api/src/main/java/io/fluo/api/data/RowColumn.java

```diff
@@ -111,12 +111,12 @@ public boolean equals(Object o) {
 
   @Override
   public int hashCode() {
-    return row.hashCode() + col.hashCode();
+    return row.hashCode() + 31 * col.hashCode();
```

Comment:
Just the bytes hash codes functions do this (because they are following the pattern in String hashcode function).  So just continuing the pattern in the underlying hashcode functions.  I think it will yield better hash codes that just adding.